examples/nlp/token_classification/conf/punctuation_capitalization_lexical_audio_config.yaml

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Punctuation and capitalization lexical audio model with pretrained BERT-like models and Encoder-Decoder-like models.
+pretrained_model: null # pretrained Punctuation and Capitalization Lexical Audio model from list_available_models(), for example:
+#
+# or your_model.nemo
+trainer:
+  devices: -1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 5
+  max_steps: -1 # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  gradient_clip_val: 0.0
+  precision: 32 # Should be set to 16 for O1 and O2, default is 16 as PT ignores it when am_level is O0
+  accelerator: gpu
+  strategy: ddp
+  enable_checkpointing: False  # Provided by exp_manager
+  logger: false  # Provided by exp_manager
+  val_check_interval: 1.0 # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step,
+  # LR schedulers, apex, etc.
+  resume_from_checkpoint: null
+  log_every_n_steps: 50
+  detect_anomaly: true
+
+exp_manager:
+  exp_dir: null  # exp_dir for your experiment, if None, defaults to "./nemo_experiments"
+  name: Punctuation_and_Capitalization_Lexical_Audio  # The name of your model
+  create_tensorboard_logger: true  # Whether you want exp_manger to create a tb logger
+  create_checkpoint_callback: true  # Whether you want exp_manager to create a model checkpoint callback
+  checkpoint_callback_params:
+    save_top_k: 3
+    monitor: "val_loss"
+    mode: "min"
+    save_best_model: true
+
+model:
+  pretrained_audio_encoder: stt_en_conformer_ctc_large # You can choose any pretrained ASR model from list_available_models() of EncDecCTCModel.
+
+  freeze_audio_encoder: false # If set to True weights of audio encoder will not be updated during training.
+
+  use_adapters: false # If set to True will enable adapters for audio encoder. Also, should provide config of adapter.
+
```

Comment:
> how about having:
> 
> ```
> audio_encoder:
>      name: stt_en_conformer_ctc_large
>      freeze: false
>      use_adapters: false
> ```
> 
> So that everything related to autoencoder in the same config field

This change probably would break all of current checkpoints as it requires changes of config structure